packages/server/src/arpa_reporter/services/fix-cell-formats.js

```diff
@@ -6,10 +6,7 @@
   dates or dollar amounts, and attempts to make them appear in Excel
   correctly formatted.
 */
-
-module.exports = {
-    fixCellFormats,
-};
+/* eslint-disable no-use-before-define */
```

Comment:
We could totally resolve this by re-ordering functions in this file, but it seemed unnecessarily noisy to do that all in this pull request.